src/v2/styles/MessageList/MessageList-layout.scss

```diff
@@ -37,6 +36,23 @@
   }
 }
 
+// React SDK does not have .str-chat__main-panel-inner element btw main-panel and list
+.str-chat__main-panel > .str-chat__list {
+  @include utils.scrollable;
+  width: 100%;
+  height: 100%;
+  max-height: 100%;
+
+  .str-chat__message-list-scroll {
+    @include utils.message-list-spacing;
+  }
+
+  .str-chat__list__center {
+    @include utils.flex-row-center;
+    width: 100%;
+  }
```

Comment:
@MartinCupela doesn't React needs the rules defined for `str-chat__ul` and so on (defined above)?

Reply:
yes, will add, thank you :)